lib/std/debug.zig

```diff
@@ -1686,7 +1686,7 @@ pub const ModuleDebugInfo = switch (native_os) {
             };
         }
     },
-    .linux, .netbsd, .freebsd, .dragonfly, .openbsd => struct {
+    .linux, .netbsd, .freebsd, .dragonfly, .openbsd, .haiku => struct {
```

Comment:
You won't get any meaningful stack trace until `getModuleForAddress` is implemented using `get_next_image_info`.

Reply:
Good point, that has been an item I have been planning to get to (most likely as a separate item).